tox.ini

```
@@ -49,12 +49,10 @@ deps =
     {[common]deps}
 
 [testenv:format]
-allowlist_externals = ./format.sh
 deps =
-    black < 24.1.0
-    reorder-python-imports
+    ruff
 commands =
-    ./format.sh []
+    ruff format []
```

This will run the isort equivalent, but it'll break `--check`. Maybe resurrect `format.sh` to handle that?
```suggestion
    ruff format []
    ruff check --fix
```

---

yeah, you could run "ruff check --diff" to figure out if there is a need to resort includes without doing that actually. I'll update the PR later. first other things.